pkg/reconciler/testing/factory.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2019 The Knative Authors.
```

Comment:
what happened to the formatting of the header?

Reply:
Looks like I lost a few newlines, will fix